### PR TITLE
Mitigate self-XSS

### PR DESCRIPTION
**Pull request by whit537**
https://hackerone.com/reports/76307

### CODE REVIEW

**Comment by whit537** [comment]
Self-merging (low-risk) security issue. I borrowed this solution from https://www.tilt.com/.

**Comment by whit537** [comment]
(Once Travis is green ...)

**Comment by whit537** [comment]
Oh heck I'll give it until tomorrow. Want to merge this, @rorepo @rohitpaulk @kzisme @techtonik et al.? :-)

**Comment by whit537** [comment]
(We're not going to have a social engineering meltdown before tomorrow.)

File: js/gratipay.js [hunk 1]
```diff
@@ -11,12 +11,24 @@ Gratipay.getCookie = function(key) {
...
 Gratipay.init = function() {
+    Gratipay.warnOffUsersFromDeveloperConsole();
     Gratipay.forms.initCSRF();
     Gratipay.signIn();
     Gratipay.signOut();
     Gratipay.payments.initSupportGratipay();
 };
 
+Gratipay.warnOffUsersFromDeveloperConsole = function()
+{
```
Comment (rohitpaulk) [inline js/gratipay.js]: For the sake of consistency, we should place this `{` on the same line as `function()`

Comment (whit537) [inline js/gratipay.js]: Fair enough. Done in 3d6eb6c062fbe22fa0361b9af5f10ab634f36a77.

**Comment by whit537** [comment]
Thanks! :-)

!m @rohitpaulk

**Pull request merged**